.gitignore

```diff
@@ -11,3 +11,4 @@ tsconfig.tsbuildinfo
 tmp/
 dist/
 .vscode/
+.yarn/cache
```

Comment:
Sorry, we are weird, but revert this. And check in the changes of `yarn/cache` folder :)

Otherwise CI breaks, it wants the cached entries (faster CI).